### PR TITLE
[14.0] delivery_auto_refresh: migration to v14

### PR DESCRIPTION
**Pull request by ilyasProgrammer**
based on  https://github.com/OCA/delivery-carrier/pull/466
This one is replacement of https://github.com/OCA/delivery-carrier/pull/472

### CODE REVIEW

**Comment by ilyasProgrammer** [comment]
@chienandalu @simahawk

**Review by chienandalu — COMMENTED** [review]
Thanks @ilyasProgrammer Tested functionaly and working Can you squash @simahawk migration commits into the single one?

**Comment by ilyasProgrammer** [comment]
> Thanks @ilyasProgrammer Tested functionaly and working Can you squash @simahawk migration commits into the single one?

@chienandalu All commits of this branch? Or maybe you could specify range of commits hashes to squash because im not sure which has to be sqashed.

**Comment by simahawk** [comment]
you don't have to squash my commits: they are improvements :wink: 
If you have to touch commits pls rewrite the last one to something like:
```
develiry_auto_refresh: enable via config params

Added parameter set_default_carrier to avoid conflicts with other modules.
```

File: delivery_auto_refresh/data/ir_config_parameter.xml [hunk 1]
```diff
@@ -2,6 +2,10 @@
 <!-- Copyright 2018 Tecnativa - Pedro M. Baeza
      License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
 <odoo noupdate="1">
+    <record id="set_default_carrier" model="ir.config_parameter">
```
Comment (simahawk) [inline delivery_auto_refresh/data/ir_config_parameter.xml:5]: this param should be set from res.config.settings IMO but as other params here are not.... I understand if you don't do it ;)

**Comment by ilyasProgrammer** [comment]
@simahawk @chienandalu  So should I squash commits or not? Which ones should I squash?

**Comment by simahawk** [comment]
From my POV no need to squash, just rewrite your last commit :pray:

**Comment by chienandalu** [comment]
:+1:

**Review by simahawk — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]

**Comment by chienandalu** [comment]
> /ocabot merge minor

Shouldn't it be nobump?

**Comment by simahawk** [comment]
> > /ocabot merge minor
> 
> Shouldn't it be nobump?

there were several changes. IMO it's preferable to "tag" those changes in another version (which BTW is better if someone was already using 14.0.1.0.0).

**Pull request merged**